frame/support/src/traits.rs

```diff
@@ -1490,6 +1490,12 @@ pub mod schedule {
 	/// higher priority.
 	pub type Priority = u8;
 
+	#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug)]
+	pub enum DelayedDispatchTime<BlockNumber> {
```

Comment:
missing doc.

Reply:
If you don't find it ambiguous, then an `impl From<BlockNumber> for DelayedDispatchTime` could be handy (which maps to `At`).

Reply:
Also, I don't agree with `Delayed` prefix. You could use `At(some_block_in_the_past)`. I think simply `DispatchTime ` is a better name.